test/boost/chunked_managed_vector_test.cc

```diff
@@ -278,11 +279,11 @@ SEASTAR_TEST_CASE(test_correctness_when_crossing_chunk_boundary) {
             for (size_t i = 0; i < (max_chunk_size + 1); i++) {
                 v.push_back(i);
             }
-            BOOST_REQUIRE(v.back() == max_chunk_size);
-            BOOST_REQUIRE(v.at(max_chunk_size) == max_chunk_size);
+            BOOST_REQUIRE(std::cmp_equal(v.back(), max_chunk_size));
```

Comment:
Lets change the type of `v` to `lsa::chunked_managed_vector<size_t>`.

Reply:
sure.